src/xo-app/vm/tab-disks.js

```diff
@@ -50,6 +51,127 @@ import {
   subscribeResourceSets,
 } from 'xo'
 
+const COLUMNS_VM_PV = [
+  {
+    itemRenderer: vdi => (
+      <Text
+        value={vdi.name_label}
+        onChange={value => editVdi(vdi, { name_label: value })}
+      />
+    ),
+    name: _('vdiNameLabel'),
+    sortCriteria: 'name_label',
+    default: true,
+  },
+  {
+    itemRenderer: vdi => (
+      <Text
+        value={vdi.name_description}
+        onChange={value => editVdi(vdi, { name_description: value })}
+      />
+    ),
+    name: _('vdiNameDescription'),
+    sortCriteria: 'name_description',
+  },
+  {
+    itemRenderer: vdi => (
+      <Size
+        value={vdi.size || null}
+        onChange={size => editVdi(vdi, { size })}
+      />
+    ),
+    name: _('vdiSize'),
+    sortCriteria: 'size',
+  },
+  {
+    itemRenderer: (vdi, userData) => {
+      const sr = userData.srs[vdi.$SR]
+      return (
+        sr !== undefined && (
+          <XoSelect
+            labelProp='name_label'
+            onChange={sr => migrateVdi(vdi, sr)}
+            predicate={sr => sr.$pool === userData.vm.$pool && isSrWritable(sr)}
+            useLongClick
+            value={sr}
+            xoType='SR'
+          >
+            <Link to={`/srs/${sr.id}`}>{sr.name_label}</Link>
+          </XoSelect>
+        )
+      )
+    },
+    name: _('vdiSr'),
+  },
+  {
+    itemRenderer: (vdi, userData) => {
+      const vbd = find(userData.vbds, vbd => vbd.id === vdi.$VBDs[0])
```

Comment:
What if a VDI is attached to more than 1 VM? How do you know it will be attached with the first VBD?